src/5e-SRD-Monsters.json

```diff
@@ -7402,20 +7670,29 @@
         "desc": "The balor makes two attacks: one with its longsword and one with its whip.",
         "options": {
           "choose": 1,
-          "from": [
-            [
-              {
-                "name": "Longsword",
-                "count": 1,
-                "type": "melee"
-              },
-              {
-                "name": "Whip",
-                "count": 1,
-                "type": "melee"
+          "type": "action",
```

Comment:
I wonder for these kind of multiattacks, if there is a better way to show this that isn't a choice.

Reply:
So I mean for Multiattacks where there is only one choice, that instead of using the choice structure, just have an `attacks` or `actions` field that is a list of the actions that you would normally put in the choice structure.

Reply:
Would something like this do?
```json
"actions": [
      {
         "name": "Multiattack",
         "desc": "The aboleth makes three tentacle attacks.",
         "actions": [
            {
               "action_name": "Tentacle",
               "count": 3,
               "type": "melee"
            }
         ]
      },
   ]
```

Also, for consistency's sake, should I change `options` to `actions` for both multi-attacks with choices and multi-attacks without choices?

Reply:
I went through with the change. I decided to use a string field `multiattack_type` to differentiate the different multiattacks.